test/transcoding/registerallocmode.ll

```diff
@@ -0,0 +1,74 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -spirv-text %t.bc -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
```

Comment:
Okay, seems like the validator doesn't handle UserSemantic properly, good for us :)

Reply:
:)